index.js

```diff
@@ -1,88 +1,119 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+var counter = 1
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+counter = 2
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+const name = "Jahaira"
+console.log(name)
 
-// console.log(name)
-// => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+// let currentThirstLevel = 5; 
+
+function drinkWater(currentThirstLevel) {
+  if (currentThirstLevel < 15) {
+    return "Man I sure an thirsty"}
+    else {
+    return " Ahh that hits the spot"
+  }
+}
```

Comment:
Nice work on structuring this function but this does not create the desired output. The goal is to have
// => "Man I sure am thirsty"
// => "Ahh that hits the spot"
// => 11
All three of the above lines print and our source is the ruby method.
def drink_water(current_thirst_level)
  puts "Man I sure am thirsty"
  current_thirst_level -= 1
  puts "Ahh that hits the spot"
  current_thirst_level
end

First thing we can do is setup the envelope for the function which you did perfectly

function drinkWater(currentThirstLevel) {
}

Next we need to account for the "puts' in ruby.  Remember that puts is just printing something to the console, it is not a return value. The equivalent to puts in js is console.log(). So we can do:
console.log("Man I sure am thirsty")

Next we need to account for "current_thirst_level -= 1" and decrement or lower the value of that variable by one. This line will be nearly identical in JS except instead of using snake_case like we do in Ruby, we are going to use camelCase
 currentThirstLevel -= 1

Next we will handle the second puts
console.log("Ahh that hits the spot")

Finally, we will return the currentThirstLevel.In  ruby, the last line of the method will automatically be returned. This is not the case in JS so we need to use the word 'return'

return currentThirstLevel

Putting it altogether:
function drinkWater(currentThirstLevel) {
  console.log("Man I sure am thirsty")
  currentThirstLevel-= 1
  console.log("Ahh that hits the spot")
  return currentThirstLevel
}